### PR TITLE
Expose metrics for profitability configuration at runtime

### PR DESCRIPTION
**Pull request by fab-10**
```
# HELP linea_profitability_conf Profitability configuration values at runtime
# TYPE linea_profitability_conf gauge
linea_profitability_conf{field="fixed_cost_wei",} 2.0E9
linea_profitability_conf{field="variable_cost_wei",} 1.0E9
linea_profitability_conf{field="eth_gas_price_wei",} 1.0E9
```

### CODE REVIEW

**Review by cloudspores — COMMENTED** [review]
Do we need to do the same for the new eth gas price now being pulled out of extradata for 3D pricing metrics?

File: sequencer/src/main/java/net/consensys/linea/extradata/LineaExtraDataPlugin.java [hunk 1]
```diff
@@ -97,5 +99,18 @@ public void start() {
...
+
+  private void initMetrics(final LineaProfitabilityConfiguration lineaProfitabilityConfiguration) {
+    final var confLabelledGauge =
+        metricsSystem.createLabelledGauge(
+            LineaMetricCategory.PROFITABILITY,
+            "conf",
+            "Profitability configuration values at runtime",
+            "field");
+    confLabelledGauge.labels(lineaProfitabilityConfiguration::fixedCostWei, "fixed_cost_wei");
+    confLabelledGauge.labels(lineaProfitabilityConfiguration::variableCostWei, "variable_cost_wei");
```
Comment (cloudspores) [inline sequencer/src/main/java/net/consensys/linea/extradata/LineaExtraDataPlugin.java:114]: Do we need to do the same for extraData content eth gas price?

Comment (fab-10) [reply sequencer/src/main/java/net/consensys/linea/extradata/LineaExtraDataPlugin.java:114]: waiting for this https://github.com/Consensys/protocol-misc/issues/953#issuecomment-2431695558 to understand what we mean by _gas price_

Comment (fab-10) [reply sequencer/src/main/java/net/consensys/linea/extradata/LineaExtraDataPlugin.java:114]: done, metrics exposed for eth gas price as well

**Review by cloudspores — APPROVED** [review]

**Pull request merged**